docker-compose.yaml

```diff
@@ -101,6 +101,7 @@ services:
       OPENFGA_AUTHN_PRESHARED_KEYS: "jimm"
       OPENFGA_DATASTORE_ENGINE: "postgres"
       OPENFGA_DATASTORE_URI: "postgresql://jimm:jimm@db/jimm?sslmode=disable"
+      OPENFGA_LOG_LEVEL: "warn"
```

Comment:
why this?

Reply:
It reduces the noise after running `docker compose up`, hiding the health check pings that happen every few seconds.